From 723a0425a399ca5afc04331886a15a5257f69b2f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 28 Feb 2021 04:31:02 +0100
Subject: [PATCH] Makefile: Don't run bootstrap-test on bootstrap by default

By default tests are run automatically after bootstrap. Normally, Alpine
tests are invoked separately in a check stage. To allow doing that this
patch disable the automatic test invocations during bootstraping.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 212c3bd3..d3c008a0 100644
--- a/Makefile
+++ b/Makefile
@@ -143,7 +143,7 @@ install-libs:
 .PHONY: bootstrap bootstrap-build bootstrap-racket bootstrap-racket-build bootstrap-test bootstrap-clean
 
 # Bootstrapping using SCHEME
-bootstrap: bootstrap-build bootstrap-test
+bootstrap: bootstrap-build
 
 bootstrap-build: support
 	cp support/c/${IDRIS2_SUPPORT} bootstrap/idris2_app
